package/chessmarkable/package

```diff
@@ -0,0 +1,27 @@
+# vim: set ft=sh:
```

Comment:
should be the env bash line

Reply:
Didn't see that we changed that. Good catch!